AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Models/MFAPreference.swift

```diff
@@ -39,11 +39,11 @@ extension MFAPreference {
             return .init(enabled: false)
         }
     }
-    
-    var softwareTokenSetting: CognitoIdentityProviderClientTypes.SoftwareTokenMfaSettingsType? {
+
+    func softwareTokenSetting(isCurrentlyPreferred: Bool? = nil) -> CognitoIdentityProviderClientTypes.SoftwareTokenMfaSettingsType {
```

Comment:
Any reason not to make this `isCurrentlyPreferred: Bool = false` like you did with `smsSetting`?

Reply:
Ahh.. Sorry.. Missed that one.. Fixed.